source/bricklaunch

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#
+# bricklaunch.sh <ops> <program>
+# - open a new vt and switch to it
+# - give 'user' ownership of the vt
+# - run the given programm on the new vt as 'user'
+#
+
+OPTIND=1
+user='user'
+openvt_opts=""
+rm_ttyfile=0
+
+function show_help {
+  cat <<END_OF_HELP
+bricklaunch.sh <opts> <program>
+
+  -h        this help
+  -s        switch to the new virtual terminal
+  -t <file> file to write the path of the new tty to
+  -u <user> user to run the program under
+  -w        wait for the program to finish
+END_OF_HELP
```

Comment:
Another option that I think would be useful it to redirect stdout and stderr to the current tty. This way you can run programs from a remote terminal and see the output on the remote terminal even though the program is actually running on a virtual console on the EV3.

I'm not asking you to implement it. I just want to know what you think about the idea.